ec2/InstanceType.dhall

```diff
@@ -293,10 +313,22 @@ let alternatives =
           =   λ(_params : Parameters.Type)
             → λ(instanceType : InstanceType)
             → merge
-                { c5-large = [ InstanceType.c5n-large ] : List InstanceType
-                , c5-xlarge = [ InstanceType.c5n-xlarge ] : List InstanceType
-                , c5-2xlarge = [ InstanceType.c5n-2xlarge ] : List InstanceType
-                , c5-4xlarge = [ InstanceType.c5n-4xlarge ] : List InstanceType
+                { c5-large =
+                      [ InstanceType.c5d-large, InstanceType.c5n-large ]
+                    : List InstanceType
+                , c5-xlarge =
+                      [ InstanceType.c5d-xlarge, InstanceType.c5n-xlarge ]
+                    : List InstanceType
+                , c5-2xlarge =
+                      [ InstanceType.c5d-2xlarge, InstanceType.c5n-2xlarge ]
+                    : List InstanceType
+                , c5-4xlarge =
+                      [ InstanceType.c5d-4xlarge, InstanceType.c5n-4xlarge ]
+                    : List InstanceType
+                , c5d-large = [ InstanceType.c5n-large ] : List InstanceType
```

Comment:
So this is why I have the permit losing disk parameter - if a workload specifies c5d because it actually requires the local disk, trying to specify c5n as an alternative will cause an outage as c5n doesn't have the local disk. So the consumer has to explicitly say that that's OK in order to get c5n as an alternative.